src/microbe_stage/AgentProjectile.cs

```diff
@@ -22,6 +22,8 @@ public class AgentProjectile : RigidBody, ITimedLife, IEntity
 
     public AliveMarker AliveMarker { get; } = new();
 
+    public string InspectableName => Properties?.ToString() ?? TranslationServer.Translate("N_A");
```

Comment:
This should probably have json ignore attribute on it. I know the other properties don't really have it that well in this class, but this is saved like other entities...